spec/classes/foreman_proxy__plugin__container_gateway_spec.rb

```diff
@@ -13,16 +13,46 @@
             '---',
             ':enabled: https',
             ":pulp_endpoint: https://#{facts[:fqdn]}",
-            ':sqlite_db_path: /var/lib/foreman-proxy/smart_proxy_container_gateway.db'
+            ':sqlite_db_path: /var/lib/foreman-proxy/smart_proxy_container_gateway.db',
```

Comment:
is this still expected in the config when using `$database_backend = 'postgres'` ? naively, I would expect the sqlite params in the template to move under the corresponding conditional

Reply:
Yeah it is -- in order to do the migration, you still need this set.

Reply:
I figured there's probably a good reason for it, thanks for the hint ;-)